Assets/UI/actionpanel.lua

```diff
@@ -1,1497 +0,0 @@
-local needBeliefTip                   :string = Locale.Lookup("LOC_ACTION_PANEL_NEEDS_BELIEF_TOOLTIP");
-local giveInfluenceTokenString        :string = Locale.Lookup("LOC_ACTION_PANEL_GIVE_INFLUENCE_TOKEN");
-local giveInfluenceTokenTip           :string = Locale.Lookup("LOC_ACTION_PANEL_GIVE_INFLUENCE_TOKEN_TOOLTIP");
-local claimGreatPersonString          :string = Locale.Lookup("LOC_ACTION_PANEL_CLAIM_GREAT_PERSON");
-local claimGreatPersonTip             :string = Locale.Lookup("LOC_ACTION_PANEL_CLAIM_GREAT_PERSON_TOOLTIP");
-local unitsHaveMovesString            :string = Locale.Lookup("LOC_ACTION_PANEL_UNIT_MOVES_REMAINING");
-local unitsHaveMovesTip               :string = Locale.Lookup("LOC_ACTION_PANEL_UNIT_MOVES_REMAINING_TOOLTIP");
-local chooseEscapeRouteString         :string = Locale.Lookup("LOC_ACTION_PANEL_CHOOSE_ESCAPE_ROUTE");
-local chooseEscapeRouteTip            :string = Locale.Lookup("LOC_ACTION_PANEL_CHOOSE_ESCAPE_ROUTE_TOOLTIP");
-local chooseDragnetPriorityString     :string = Locale.Lookup("LOC_ACTION_PANEL_CHOOSE_DRAGNET_PRIORITY");
-local chooseDragnetPriorityTip        :string = Locale.Lookup("LOC_ACTION_PANEL_CHOOSE_DRAGNET_PRIORITY_TOOLTIP");
-local needArtifactPlayerString        :string = Locale.Lookup("LOC_ACTION_PANEL_CHOOSE_ARTIFACT_PLAYER");
-local needArtifactPlayerTip           :string = Locale.Lookup("LOC_ACTION_PANEL_CHOOSE_ARTIFACT_PLAYER_TOOLTIP");
-local cityRangedAttackString          :string = Locale.Lookup("LOC_ACTION_PANEL_CITY_RANGED_ATTACK");
-local cityRangedAttackTip             :string = Locale.Lookup("LOC_ACTION_PANEL_CITY_RANGED_ATTACK_TOOLTIP");
-local cloudTurnUploadingString        :string = Locale.Lookup("LOC_ACTION_PANEL_CLOUD_TURN_UPLOADING");
-local cloudTurnUploadingTip           :string = Locale.Lookup("LOC_ACTION_PANEL_CLOUD_TURN_UPLOADING_TOOLTIP");
-local cloudTurnUploadedString         :string = Locale.Lookup("LOC_ACTION_PANEL_CLOUD_TURN_UPLOADED");
-local cloudTurnUploadedTip            :string = Locale.Lookup("LOC_ACTION_PANEL_CLOUD_TURN_UPLOADED_TOOLTIP");
-local yourTurnToolStr                 :string = Locale.Lookup("LOC_KEY_YOUR_TURN_TIME_TOOLTIP");
-local estTilTurnToolStr               :string = Locale.Lookup("LOC_KEY_ESTIMATED_TIME_TIL_YOUR_TURN_TIME_TOOLTIP");
-local estTimeElapsedToolStr           :string = Locale.Lookup("LOC_KEY_ESTIMATED_TIME_ELAPSED_TOOLTIP");
-local canUnreadyTurnTip               :string = Locale.Lookup("LOC_ACTION_PANEL_CAN_UNREADY_TOOLTIP");
--- ==== CQUI CUSTOMIZATION BEGIN  ==================================================================================== --
-local cqui_encampmentRangedAttackString    :string = Locale.Lookup("LOC_CQUI_ACTION_PANEL_ENCAMPMENT_RANGED_ATTACK");
-local cqui_encampmentRangedAttackTip       :string = Locale.Lookup("LOC_CQUI_ACTION_PANEL_ENCAMPMENT_RANGED_ATTACK_TOOLTIP");
--- ==== CQUI CUSTOMIZATION END ======================================================================================= --
```

Comment:
There seem to be CQUI customizations in this file. Haven't looked closely just did a diff and saw stuff.

Reply:
ah you're right.  I swear I checked it multiple times - but I must've pasted the wrong file multiple times (I use notepad++ to do compares since it can ignore spaces and tabs better than VSCode does).  I am undeleting it now